sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
Then the table should be made more generic for other hardware rewrites which we have quite a lot of in arrays already. At the moment this is too specific imho.

@librenms/reviewers thoughts?